### PR TITLE
Setup AppTextInput component

### PR DESCRIPTION
**Pull request by mhevyk**

### CODE REVIEW

**Comment by RossWebFS** [comment]
Don't forget to connect this pr with #243

File: src/styles/app-text-input/AppTextInput.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    .app-input__control-container {
+      align-self: stretch;
+    }
+  }
+
+  &__filled {
+    border: 1px solid black;
+  }
+
+  &__underlined {
```
Comment (RossWebFS) [inline src/styles/app-text-input/AppTextInput.scss:54]:
```suggestion
    &__underlined {
    transition: all 0.3s ease;
    border: none;
    border-bottom: 1px solid black;

    &:focus-visible {
      outline: none;
      transform: scaleX(1.05);
    }
  }
```

image

The underlined variant doesn't apply any styles on focusing. I think, we can ask out designed to create styles for focus (or I suggest to use my variant of animation)

File: src/styles/app-text-input/AppTextInput.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    border: none;
+    border-bottom: 1px solid black;
+
+    &:focus-visible {
+      outline: none;
+    }
+  }
+
+  &__helper-text {
+    color: #aaa;
```
Comment (RossWebFS) [inline src/styles/app-text-input/AppTextInput.scss:64]: I added file for variables in pr #241. We can move colors to that folder

Comment (mhevyk) [reply src/styles/app-text-input/AppTextInput.scss]: Fixed, also added alias for design system. Please take a look

File: src/styles/app-text-input/AppTextInput.scss (hunk 1 above)
Comment (mhevyk) [reply src/styles/app-text-input/AppTextInput.scss:57]: We need to ask designer about that. I left a todo. I guess after deploying storybook we can do more detailed testing and detect some issues

**Review by RossWebFS — APPROVED** [review]

**Pull request merged**